### ISSUE TITLE
cookies acceptation

### ISSUE THREAD

**Issue by JacquesDhuesme** [bet3-0/b3t#142]
Créer un footer pour l'acceptation des cookies

**Comment by JacquesDhuesme** [comment]
Voilà ce qu'il faut mettre avec un bouton "j'accepte"
"Ce site utilise les cookies pour vous offrir la meilleure et la plus pertinente des expériences utilisateur. Pour plus de renseignement consulter les mentions légales, rubrique cookies"

**Comment by nils-van-zuijlen** [comment]
Il faut aussi un bouton "Je refuse", avec la même apparence, on est en Europe, donc RGPD.

Et est-ce qu'on a des cookies non nécéssaires au fonctionnement ? Parce que si on n'en a pas, il n'y a pas besoin de bandeau.

**Comment by JacquesDhuesme** [comment]
Oui oui et on en utilise pour la connexion

**Comment by guiyom-e** [comment]
On n'a pas de cookies, mais des données en local storage. On ne collecte aucune donn ée personnelle. Le consentement peut être recueilli au login à la limite.

**Comment by nils-van-zuijlen** [comment]
Oui, si on n'utilise de "cookies" que pour la connexion, pas besoin de bandeau, juste d'une phrase au login, et encore... c'est du fonctionnement normal sans collecte ni traitement de données.

**Comment by thomasgall67** [comment]
Si pas de cookie, pas de bandeau, si y'a un mini cookie (pas aux noisettes, mais juste de login) faut le mettre (meme si pas besoin de paramétrer)
